src/logfile_values.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+#
+# logfile_values.py
+#
+# Copyright (c) 2017, InnoGames GmbH
+#
+"""
+logfile_values.py -- a python script to find metrics values in log file
+
+This script is using last line of log file to get metric value by column number
+
+python log_values.py --name=n` --column=1 --name=n2 --column=2
+"""
+
+from argparse import ArgumentParser
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='logfile_values')
+    parser.add_argument('--file', default='1.log')
+    parser.add_argument(
+        '-n',
+        '--name',
+        help='Graphite metric name',
+        default=None,
+        action='append',
+        dest='names', )
+    parser.add_argument(
+        '-c',
+        '--column',
+        help='Number of the column with metric value',
+        default=None,
+        action='append',
+        dest='columns', )
+    args = parser.parse_args()
```

Comment:
I don't like dependent command like arguments.  I would design it something like this:

```python
class Metric:
    def __init__(self, arg):
        if ':' not in arg:
            raise ArgumentTypeError('Argument must have ":"')
        self.name, column = arg.split(':', 1)
        if not column.isdecimal():
            raise ArgumentTypeError('Column must be a number')
        self.column = int(column)
```

```python
    parser.add_argument('metric', type=Metric, action='append')
```

`ArgumentParser` lets you use simple functions as types as well, but I think class is better for this script.  We can extend it later to get the metrics from lines other than the last one.